Sources/DangerFixtures/DangerDSLResources/DangerDLSBitBucketServer.swift

```diff
@@ -1,4 +1,4 @@
-// swiftlint:disable file_length
+// swiftlint:disable line_length
```

Comment:
<!--
  0 failure: 
  1 warning:  SwiftLint rule 'l...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerFixtures/DangerDSLResources/DangerDLSBitBucketServer.swift;
  Line: 1;
-->

- :warning: SwiftLint rule 'line_length' did not trigger a violation in the disabled region. Please remove the disable command. (`superfluous_disable_command`)